config-templates/netapp/netapp-trident/20.07/custom-parameters.json

```diff
@@ -70,6 +70,6 @@
     "displayname": "User Password",
     "name": "user-pass",
     "required": "true",
-    "type": "text"
+    "type": "secret"
```

Comment:
I think , we should add both username and password as `secret`

Reply:
how to differentiate between the filed one need obfuscation and the other one does not require obfuscation?

Reply:
change secret type to `secret`